### DISCUSSION TITLE
Question about Analytics

### DISCUSSION THREAD

**Discussion by jacob-willden** [pkgxdev/pkgx#537] [Help] [answered]
Hello tea community! Thank you so much for building and maintaining this amazing project! As I'm starting to migrate from Homebrew to Tea, I'm curious what analytics Tea collects. I know that [Homebrew uses both its own InfluxDB and Google Analytics](https://docs.brew.sh/Analytics), but I couldn't find any analytics information in the [Tea docs](https://docs.tea.xyz/) or in the site's [privacy policy](https://tea.xyz/privacy-policy/). Thanks in advance for your time and help!

**Comment by mxcl** [comment, marked as answer]
we calculate total unique users from the AWS CloudFront logs (where bottles are stored). That's it!

The website does take more analytics. I’m delegating to @owenjames about that.

We don't intend to do anything else in the developer tooling. 

Sorry we don’t specify this information anywhere. That needs fixing.

**Comment by owenjames** [comment]
For the website, it's just the common Google Analytics data collection for website views, and it's not connected with tea cli usage

**Comment by jacob-willden** [comment]
Cool! Thanks for the information!
